server/resolvers/article/queries.js

```diff
@@ -24,67 +27,78 @@ const articleQueries = {
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
         keywords: ['Lorem', 'Ipsum'],
         minutesToRead: 8
       },
       {
         id: 8,
         number: 8,
         content:
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
         keywords: ['Lorem', 'Ipsum'],
         minutesToRead: 8
       },
       {
         id: 9,
         number: 9,
         content:
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
         keywords: ['Lorem', 'Ipsum'],
         minutesToRead: 8
       }
     ]
+    const { data } = await axios.post(SEARCH_SERVICE_ENDPOINT, { query: search })
+    const articleIds = [...data.articles]
+
+    const articles = []
+    for (const id of articleIds) {
+      for (const article of database) {
+        if (id === article.id) { articles.push(article) }
+      }
+    }
+
+    return articles
```

Comment:
Are there unit tests for this?

Reply:
This is the mocked version of article by id db endpoint

Reply:
There is still application logic in this method that can be unit tested, even if the articles are mocked out. For example, it is making a `POST` request to the search endpoint.

Reply:
I'm pending to add unit tests for this endpoint. But to keep PR as concise as possible, I think is a good idea to have it as a next step on a following PR.

Reply:
Please add unit tests before merging. Code should not be merged to main without unit tests.